tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1373,6 +1373,45 @@ Status Converter::RenameAndMarkOutputTensors(
   return Status::OK();
 }
 
+#if IS_TRT_VERSION_GE(7, 1, 3, 0)
+// Force TensorRT deterministic behavior by enforcing
+// TRT tactic selection to be a specific one controlled by the env var:
+// TF_DEBUG_TRT_DETERMINISTIC_TACTIC_ID.
+class DeterministicAlgorithmSelector : public nvinfer1::IAlgorithmSelector
```

Comment:
Rename the class to StaticAlgorithmSelector,
Add a private member "int32_t algorithm_id_"
Add a constructor:
    StaticAlgorithmSelector(int32_t id) : algorithm_id_(id) {}

Reply:
Done